server/apps/dictionaries/service.py

```diff
@@ -111,8 +137,8 @@ def get_model_for_lang(self, lang):
         :param lang: language code
         """
         model = {}
-        lookup = {'$and': [{'language_id': lang}, {'is_active': {'$in': ['true', None]}}]}
-        dicts = self.get(req=None, lookup=lookup)
+        dicts = self.get_dictionaries(lang)
```

Comment:
I suggest caching dictionaries and models. This takes a lot of processing and the spelling suggestion requests should happen quite often.